index.js

```diff
@@ -142,6 +171,31 @@ class Timber {
 	create(...args) {
 		return new Timber(...args);
 	}
+
+	getDefaults() {
+		const defaults = {};
+		if (is.renderer) {
+			Object.assign(defaults, electron.remote.getGlobal(defaultsNameSpace) || {});
+		} else {
+			Object.assign(defaults, global[defaultsNameSpace]);
+		}
+
+		return defaults;
+	}
+
```

Comment:
I thought might be handy to have for debugging purposes.
This method returns a copy of the defaults, so users don't accidentally mutate them.